### PR TITLE
Provide a simple 'hello-world' template that's using profile volumes.

### PR DESCRIPTION
**Pull request by nfnt**

### CODE REVIEW

File: frameworks/helloworld/src/main/dist/profile-mount-volume.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+               echo profile >> profile-container-path/output
+               sleep $SLEEP_DURATION
+        cpus: {{HELLO_CPUS}}
+        memory: {{HELLO_MEM}}
+        env:
+          SLEEP_DURATION: {{SLEEP_DURATION}}
+          VOLUME_PROFILE: {{HELLO_VOLUME_PROFILE}}
+        volume:
+          path: profile-container-path
+          profiles: [{{HELLO_VOLUME_PROFILE}}]
```
Comment (takirala) [inline frameworks/helloworld/src/main/dist/profile-mount-volume.yml]: we have sth similar at https://github.com/mesosphere/dcos-commons/blob/9827ee6fbee1803df47345b3e1b450484142ac7d/frameworks/helloworld/src/main/dist/pod-profile-mount-volume.yml#L14 as well fwiw

Comment (nfnt) [reply frameworks/helloworld/src/main/dist/profile-mount-volume.yml]: Yes, this is based on `pod-profile-mount-volume` but uses a single profile volume instead. `pod-profile-mount-volume` also uses a regular persistent volume. This is useful for testing profile volume support in the SDK. `profile-mount-volume` is meant as a minimal configuration for testing profile volumes as part of `hello-world`.

**Comment by takirala** [comment]
@nfnt I presume you want this in universe to unblock mesosphere/dcos-storage#521 ?

**Comment by nfnt** [comment]
I'm using a stub universe created from https://github.com/mesosphere/dcos-storage/pull/521 at the moment. Therefore it isn't urgent to get this into Universe. But once it's there, I'd happily use the Universe package instead.

**Pull request merged**